mailbox/cassandra/pom.xml

```diff
@@ -129,6 +129,11 @@
             <artifactId>james-server-task-memory</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-testing</artifactId>
+            <scope>runtime</scope>
```

Comment:
scope test is better?

Reply:
The maven test module `james-server-testing` with scope `test` is not transitive to other testing modules (lead to NoDefClassFound..). 

So I use `runtime` scope: `This scope indicates that the dependency is not required for compilation, but is for execution. Maven includes a dependency with this scope in the runtime and test classpaths, but not the compile classpath.`
rf: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-scope

Better idea?